sklearn/utils/sparsefuncs.py

```diff
@@ -84,16 +91,22 @@ def mean_variance_axis(X, axis):
     """
     _raise_error_wrong_axis(axis)
 
+    if ddof < 0:
+        raise ValueError('ddof cannot be <0')
+
+    if ddof >= X.shape[axis]:
+        raise ValueError('ddof must be <N')
```

Comment:
Please explain what N is in the message, and indicate what value was passed for ddof.